db/queries.py

```diff
@@ -1,46 +1,75 @@
-from db.models import Message, User, Chat
 from django.db.models import Q, Count, F
 
+from db.models import Message, User, Chat
+
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__contains=word))
```

Comment:
Use case-insensitive containment test.

Reply:
You're right.  I have to read the paragraphs to the end